products/compute/api.yaml

```diff
@@ -301,6 +301,40 @@ objects:
               instance may take to initialize. To do this, create an instance
               and time the startup process.
             default_value: 60
+          - !ruby/object:Api::Type::Enum
+            name: 'mode'
+            default_value: :ON
+            description: |
+              Defines operating mode for this policy.
+            values:
+              - :OFF
+              - :ONLY_UP
+              - :ON
+          - !ruby/object:Api::Type::NestedObject
+            name: 'scaleDownControl'
+            min_version: beta
+            description: |
+              Defines scale down controls to reduce the risk of response latency
+              and outages due to abrupt scale-in events
+            properties:
+            - !ruby/object:Api::Type::NestedObject
+              name: 'maxScaledDownReplicas'
```

Comment:
Can you add `at_least_one_of:` for this?

It helps us prevent issues around empty blocks by requiring at least one field per block is specified.

In this case I believe it will look like:
```
at_least_one_of:
  - scale_down_control.0.max_scaled_down_replicas.0.fixed
  - scale_down_control.0.max_scaled_down_replicas.0.percent
```

And then one to `scaleDownControl` that looks like:

```
at_least_one_of:
  - scale_down_control.0.max_scaled_down_replicas
  - scale_down_control.0.time_window_sec
```

For example: https://github.com/GoogleCloudPlatform/magic-modules/blob/master/products/compute/api.yaml#L757